Packs/ForcepointDLP/Integrations/ForcepointEventCollector/ForcepointEventCollector.yml

```diff
@@ -1,86 +1,1150 @@
 category: Email
 sectionOrder:
-- Connect
-- Collect
+  - Connect
+  - Collect
 commonfields:
   id: Forcepoint DLP Event Collector
   version: -1
 configuration:
-- display: Server URL (e.g., https://<DLP Manager IP>:<DLP Manager port>/)
-  name: url
-  required: true
-  type: 0
-  section: Connect
-- display: Username
-  name: credentials
-  required: true
-  type: 9
-  section: Connect
-- defaultvalue: '10000'
-  display: Maximum number of events per fetch
-  name: max_fetch
-  required: false
-  type: 0
-  section: Collect
```

Comment:
This value is used also for fetch-events in XSIAM? If so we should split it... this is not the same parameter for fetching incidents and events...

Reply:
I see that it is being used:
```python
        elif command == "fetch-incidents":
            max_fetch=arg_to_number(params.get("max_fetch")) or DEFAULT_LIMIT
```